### ISSUE TITLE
PDF: Wrong fonts used when generated on GHA

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/bipm-si-brochure#188]
In the site artifact generated by https://github.com/metanorma/bipm-si-brochure/actions/runs/2997291371 , the fonts used for the SI Brochure generated through the `collection` command (English, French and bilingual) are wrong:

* `collection.pdf`
* `collection_en.pdf`
* `collection_fr.pdf`

<img width="621" alt="Screen Shot 2022-09-13 at 12 34 57 PM" src="https://user-images.githubusercontent.com/11865/189809581-266f8472-a387-44c3-b330-b9bf6c235841.png">

<img width="591" alt="Screen Shot 2022-09-13 at 12 35 12 PM" src="https://user-images.githubusercontent.com/11865/189809614-692d7038-8030-403c-b238-21e866968792.png">

However, the direct documents generated have correct fonts:
* `si-brochure-fr.pdf`
* `si-brochure-en.pdf`

This file contains `collection*.pdf` and `collection*.xml` for debugging:
[Archive.zip](https://github.com/metanorma/bipm-si-brochure/files/9553550/Archive.zip)

**Comment by Intelligent2013** [comment]
@ronaldtse the parameter `--font-manifest ...` is missing in the command line for collection PDF processing:
```
java -Xss5m -Xmx2048m -jar /usr/local/bundle/gems/mn2pdf-1.50/lib/../bin/mn2pdf.jar --xml-file "collection/collection.presentation.xml" --xsl-file "/usr/local/bundle/gems/metanorma-bipm-2.1.7/lib/isodoc/bipm/bipm.brochure.xsl" --pdf-file "collection/collection.pdf" --syntax-highlight  --split-by-language 
```
therefore mn2pdf puts default Noto fonts.

For instance, there is `-font-manifest ...` in the command line for usual documents:
```
java -Xss5m -Xmx2048m -jar /usr/local/bundle/gems/mn2pdf-1.50/lib/../bin/mn2pdf.jar --xml-file "/__w/bipm-si-brochure/bipm-si-brochure/site/documents/si-brochure-en.presentation.xml" --xsl-file "/usr/local/bundle/gems/metanorma-bipm-2.1.7/lib/isodoc/bipm/bipm.brochure.xsl" --pdf-file "/__w/bipm-si-brochure/bipm-si-brochure/site/documents/si-brochure-en.pdf" --param baseassetpath="/__w/bipm-si-brochure/bipm-si-brochure/sources" --syntax-highlight  --font-manifest "/tmp/fontist_locations20220906-1233-wyxn8g.yml"
```

A 1.5 years ago, we've had the similar issue: https://github.com/metanorma/bipm-si-brochure/issues/108#issuecomment-816585382

@CAMOBAP could you check it? Thanks.

**Comment by ronaldtse** [comment]
Then this could be an issue with the new "collection" command. It must be failing to properly inherit those parameters like the "site generate" command. @abunashir can you please help here? This is important. Thanks.

**Comment by ronaldtse** [comment]
@abunashir is this fixed? Thanks!

**Comment by abunashir** [comment]
Nope, not yet. I am looking into it next!

**Comment by abunashir** [comment]
Hi @ronaldtse, I finally managed to drill down the issue. Looks like the `font-mainfest` was never being used for collection document, so even though we were trying to set it but the pdf generation was ignoring that. I have created two separate pull request, and once that is merged then this issue should be resolved. Here are pull request:

- https://github.com/metanorma/metanorma-standoc/pull/699
- https://github.com/metanorma/metanorma/pull/265

I've also tried to use this edge version [here](https://github.com/metanorma/bipm-si-brochure/pull/186/commits/478433bd88321e18ba7d60cfe141f5c27c087429), and it seems to be working as expected :)

<img width="1124" alt="Screenshot 2022-11-06 at 15 05 50" src="https://user-images.githubusercontent.com/1220911/200175817-838a81a3-e471-419d-a220-d322d461d1c1.png">


//cc: @opoudjis, @CAMOBAP

**Comment by abunashir** [comment]
@ronaldtse: I assume we are done with this one, can we close this?

**Comment by abunashir** [comment]
Closing this one, but if there is anything else please feel free to reopen this one
